bindings/ros/aditof_roscpp/src/aditof_utils.cpp

```diff
@@ -0,0 +1,125 @@
+#include "aditof_utils.h"
+
+using namespace aditof;
+std::shared_ptr<Camera> initCameraEthernet(int argc, char **argv) {
+    google::InitGoogleLogging(argv[0]);
+    FLAGS_alsologtostderr = 1;
+    Status status = Status::OK;
+
+    if (argc < 2) {
+        LOG(ERROR) << "No ip provided!";
+        return std::shared_ptr<Camera>(nullptr);
+    }
+
+    std::string ip = argv[1];
+
+    System system;
+    status = system.initialize();
+    if (status != Status::OK) {
+        LOG(ERROR) << "Could not initialize system!";
+        return std::shared_ptr<Camera>(nullptr);
+    }
+
+    std::vector<std::shared_ptr<Camera>> cameras;
+    system.getCameraListAtIp(cameras, ip);
+    if (cameras.empty()) {
+        LOG(WARNING) << "No cameras found";
+        return std::shared_ptr<Camera>(nullptr);
+    }
+
+    std::shared_ptr<Camera> camera = cameras.front();
+    status = camera->initialize();
+    if (status != Status::OK) {
+        LOG(ERROR) << "Could not initialize camera!";
+        return std::shared_ptr<Camera>(nullptr);
+    }
+
+    std::vector<std::string> frameTypes;
+    camera->getAvailableFrameTypes(frameTypes);
+    if (frameTypes.empty()) {
+        ROS_INFO("no frame type avaialble!\n");
```

Comment:
Why info and not error?

Reply:
My mistake. I should have used the glog logging statements instead of the ones used in ros. Fixed it, thanks